src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.component.html

```diff
@@ -169,7 +261,8 @@
 
     <!-- Row shown when there is no matching data. -->
     <tr class="mat-row" *matNoDataRow>
-        <td class="mat-cell text-center" colspan="4">No data matching the filter</td>
+        <td *ngIf="!accountView" class="mat-cell text-center" colspan="4">No data matching the filter</td>
+        <td *ngIf="accountView" class="mat-cell text-center" colspan="5">No data matching the filter</td>
```

Comment:
Can't you just check the size of `DISPLAY_COLUMNS` ?

Reply:
Fixed, added [attr.colspan]=...